src/actionCreators/resources.js

```diff
@@ -187,7 +190,6 @@ export const expandProperty = (propertyKey, errorKey) => (dispatch, getState) =>
 export const contractProperty = (propertyKey) => (dispatch, getState) => {
   const property = selectProperty(getState(), propertyKey)
   property.values = null
-  dispatch(removeProperty(propertyKey))
```

Comment:
Justin L and I determined this is not needed

Reply:
`addProperty` updates the property, so no need to remove it first.

Reply:
doesn't look it, just ditched and the tests all passed still